addons/payment_mercado_pago/const.py

```diff
@@ -30,7 +30,7 @@
 # Mapping of transaction states to Mercado Pago payment statuses.
 # See https://www.mercadopago.com.mx/developers/en/reference/payments/_payments_id/get.
 TRANSACTION_STATUS_MAPPING = {
-    'pending': ('pending', 'in_process', 'in_mediation'),
+    'pending': ('pending', 'in_process', 'in_mediation', 'authorized'),
     'done': ('approved', 'refunded'),
-    'canceled': ('cancelled', 'null'),
+    'canceled': ('cancelled', 'null', 'rejected'),
```

Comment:
Maybe map this status to 'error' then, and explicitly manage that case in `_process_notification_data` =]

Reply:
@Demesmaeker I am fine with adding the error status. 
Add human readable errors for the case of rejected. 
if you are ok with the change  I will do  rebase,